app/notifications/validators.py

```diff
@@ -159,7 +157,8 @@ def validate_and_return_extended_phone_number_info(service, send_to, key_type, c
 
         recipient_data = _get_extended_phone_number_info(phone_number, send_to)
 
-        if recipient_data["international"] and check_intl_sms_limit:
+        # UK_PREFIX includes standard domestic numbers, and crown dependency numbers
+        if str(phone_number.number.country_code) != UK_PREFIX and check_intl_sms_limit:
```

Comment:
We have [a method](https://github.com/alphagov/notifications-utils/blob/9154f4b5d0ff5d9ad212cb6f08b31297bdeeb63a/notifications_utils/recipient_validation/phone_number.py#L224-L232) for this already which reads a bit nicer:
```suggestion
        if not phone_number.is_uk_phone_number() and check_intl_sms_limit:
```